server/src/main/java/org/opensearch/cluster/routing/allocation/allocator/BalancedShardsAllocator.java

```diff
@@ -154,6 +154,13 @@ public class BalancedShardsAllocator implements ShardsAllocator {
         Property.NodeScope
     );
 
+    public static final Setting<Boolean> IGNORE_THROTTLE_FOR_REMOTE_RESTORE = Setting.boolSetting(
+        "cluster.routing.allocation.ignore_throttle_for_restore",
```

Comment:
Let's rename this better to `cluster.routing.allocation.recovery.remote_primary.ignore_balance `

Reply:
Renamed to `cluster.routing.allocation.remote_primary.ignore_throttle`